spdx-utils/src/main/kotlin/SpdxLicenseMapping.kt

```diff
@@ -19,337 +19,331 @@
 
 package com.here.ort.spdx
 
+import com.here.ort.spdx.SpdxLicense.*
+
 /**
- * A mapping from license strings collected from the declared licenses of open source packages to SPDX license
- * identifiers.
+ * A mapping from license strings collected from the declared licenses of open source packages to SPDX expressions.
  */
 object SpdxLicenseMapping {
     private val mapping = mapOf(
-            "(EDL-1.0 OR EPL-1.0)" to enumSetOf(SpdxLicense.EPL_1_0),
-            "(MIT-style) netCDF C library license" to enumSetOf(SpdxLicense.NETCDF),
-            "2-clause BSD license" to enumSetOf(SpdxLicense.BSD_2_CLAUSE),
-            "2-clause BSDL" to enumSetOf(SpdxLicense.BSD_2_CLAUSE),
-            "3-Clause BSD" to enumSetOf(SpdxLicense.BSD_3_CLAUSE),
-            "3-Clause BSD License" to enumSetOf(SpdxLicense.BSD_3_CLAUSE),
-            "3-clause BSD" to enumSetOf(SpdxLicense.BSD_3_CLAUSE),
-            "AFLv2.1" to enumSetOf(SpdxLicense.AFL_2_1),
-            "AL 2.0" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "ALv2" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "APL2" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "APLv2.0" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "ASF 2.0" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "ASL" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "ASL 2" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "ASL 2.0" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "ASL, version 2" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Academic Free License (AFL)" to enumSetOf(SpdxLicense.AFL_2_1),
-            "Apache" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache  Version 2.0, January 2004" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache 2" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache 2.0" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache 2.0 License" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache License" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache License (2.0)" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache License 2" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache License 2.0" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache License V2.0" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache License Version 2" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache License Version 2.0" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache License v2" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache License v2.0" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache License, 2.0" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache License, V2.0 or later" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache License, Version 2" to enumSetOf(SpdxLicense.APACHE_2_0),
-            "Apache License, Version 2.0" to enumSetOf(SpdxLicense.APACHE_2_0),
+            "(EDL-1.0 OR EPL-1.0)" to (BSD_3_CLAUSE or EPL_1_0),
```

Comment:
Where does BSD come from here? It was not part of the set before.

Reply:
EDL-1.0 stands for the Eclipse Distribution License which is already mapped to BSD_3_CLAUSE below.